packages/react/react/src/hooks/useCombinedRefs.spec.tsx

```diff
@@ -8,7 +8,7 @@ describe('useCombinedRefs', () => {
     const refs: Array<HTMLDivElement | null> = [];
     const callbackRef = jest.fn();
 
-    function TestComponent() {
+    const TestComponent = () => {
```

Comment:
Is it any benefit to use arrow function?
I'm curious because I couldn't find it

Reply:
@okinawaa 👋
I've seen some other test code that uses an arrow function to define "TestComponent", so I switched to that for consistency.

However, I've also seen test code written as function declarations, so there doesn't seem to be a rule. 

It doesn't seem to have a big impact, but should I change it back?